<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherAI</class>
 <widget class="QDialog" name="WeatherAI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天气预报(测试版)-天气AI</string>
  </property>
  <widget class="QPushButton" name="Button1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C++ 模型训练</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入近三天最低气温进行预测</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditone">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>133</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第一天</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第二天</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第三天</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdittwo">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditthree">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预测结果</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditfour">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
